modules/ROOT/pages/osv2-guide.adoc

```diff
@@ -113,10 +112,51 @@ Object Store v2 provides persistent storage of objects with a time to live (TTL)
 
 The tutorial provides more detailed information about creating object stores in Mule 3 and Mule 4 applications. 
 
+== Configure a Custom Time-to-Live
+
+To configure a custom time-to-live (TTL) period for keys stored in Object Store v2:
+
+. Create an app that uses Mule runtime engine version 4.2.1 or later.
+. Configure the object store in the app.
+. Select the *Store* operation in the flow and, in the properties window, click the green plus icon (+) to the right of the *Object store* field:
++
+.The arrow shows the green plus icon in the properties window.
+image::os-use-ttl-props.png[Green plus icon in the properties window]
+. Configure the TTL entries in the *Global Object Store Configuration* window:
++
+.The arrow indicates the *Entry ttl* field in the *Global Object Store Configuration* window.
+image::os-use-ttl-global-props.png[Global Object Store Configuration window]
++
+[NOTE]
+For Object Store v2, the *Max entries* field is not configurable.
++
+* *Entry ttl* 
++
+To use rolling TTL, leave this field empty.
+Otherwise, enter the custom TTL (between 0 and 2592000 seconds).
```

Comment:
The introduction to this task is how to create a CUSTOM TTL, not how to use rolling TTL, so I would not mention this or, if you feel you must, I would not make it the first sentence. Assumably, the user will NOT be leaving this field empty. It is confusing to include this; outside the scope of the topic.